packages/featureflags/react.d.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference types="react" />
```

Comment:
I want to add these, they make sense.  But I'm not a Typescript compiler.  How do we verify these are correct?

Reply:
that's the question? we'd have to have some way to validation it. maybe something exists we can integrate?